src/Minimap/Minimap.js

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useMemo } from 'react';
```

Comment:
folder name to `minimap`

Reply:
I tried changing the folder name simply in the previous commit only by simply renaming the folder. But it didn't work. Then I used `git config core.ignorecase false` and then committed. It did work on my local machine, but when I pushed the code, it now has 2 folders `Minimap` and `minimap`. I can not delete it as it is not the case in my local machine even after pulling origin. What can I do?